### ISSUE TITLE
Add the Ability to use multiple explorers/daemons to j.clients.blockchain.rivine

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldtech/jumpscale_lib#80]
Ha for light clients is achieved by using multiple explorers/daemons. In case one  is down another  one can be used.

**Comment by ahussein** [comment]
fixed by 432053

**Comment by ahussein** [comment]
fixed by 432053e

**Comment by Pishoy** [comment]
```In [13]: seed = j.clients.rivine.generate_seed()

In [14]: from JumpscaleLib.clients.blockchain.rivine.RivineWallet import RivineWallet

In [15]: wallet = RivineWallet(seed=seed, bc_networks=['https://exploer-unreachable.com','https://explorer.testnet.threefoldtoken.com/'],bc_network_password='test123',nr_keys_per_seed=5,minerfee=100000000)

In [16]:         wallet.current_balance
[Thu06 11:38] - utils.py          :89  :lockchain.rivine.utils - WARNING  - Failed to get current chain height.
[Thu06 11:38] - utils.py          :174 :lockchain.rivine.utils - WARNING  - Failed to retrieve unconfirmed transactions.
[Thu06 11:38] - RivineWallet.py   :149 :in.rivine.rivinewallet - INFO     - Current chain height is: 107900
Out[16]: 0.0

In [17]: wallet.addresses
Out[17]: 
['01d655d5f5f2109fe8f9c55a10b228cc77ab81c5a8a709a14e3b480e9207323a43bed42c559fd4',
 '01382c4d2406771f534add1d7b20c82692275bab6bfdf0bdad7455e164354b30678d1e7670d106',
 '014a76b9f45cdac146ef347e506b5f1cbd91cc2d76d51267ede400ca0af4ad916455285c7265da',
 '0106f6d02da72f1b9eb19a446d425719de9c9476f54807dc31e85b2a301b97ba53934c5a696a5e',
 '013fdaea00272941020aa355bfd45e47fb17e77fa72b8286fbcef97a0f882aeb343dd335a9ae63']


```
0-templates
commit 8c5977a172919feb3019e994f576d75c8f0e68eb
Author: serboctor <boctor.sarah@gmail.com>
Date:   Wed Sep 5 16:23:16 2018 +0200

    Fix dm_vm docs

jumpscale_core
commit 49aeca1b21a3dcd1cf36e82ea065781efbe3e434
Author: Jan DeLandt <jan@bettertoken.com>
Date:   Wed Sep 5 16:17:16 2018 +0000

    cleanup and cutting over from development_dynamic
    
    keeping up-to-date is extremely challenging.  getting really fed up with it.
jumpscale_lib
commit 4b1a32c43f4ce446ef88a37846a405e6ab960076
Author: Abdelrahman Hussein <ahussein.abdelrahman@gmail.com>
Date:   Wed Sep 5 22:29:48 2018 +0200

    RivineClient: update timeout when checking for different explorer nodes
jumpscale_prefab
commit 502ffd7ede891012799a5ae87043291a761d09d0
Author: despiegk <kristof@incubaid.com>
Date:   Sun Aug 26 08:20:13 2018 +0200


```
